### PR TITLE
feat: Update matrix actions to add 'fail-fast' input

### PR DESCRIPTION
**Pull request by DariuszJerzewski**
## Description

Update matrix actions to add 'fail-fast' input. This will allow all of the matrix actions to finish even if one of them failed. 

Related issue: N/A

## Before submitting (or marking as "ready for review")

- [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [x] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

**Comment by sdh100shaun** [comment]
General comments 

You have checked Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification? - but it does not can that please be fixed 

We will need to update the readme if we adopt this approach but that is not included in the PR

**Comment by DariuszJerzewski** [comment]
> General comments
> 
> You have checked Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification? - but it does not can that please be fixed
> 
> We will need to update the readme if we adopt this approach but that is not included in the PR

PR title updated and made the change in README to include the new variable.

File: README.md [hunk 1]
```diff
@@ -543,4 +554,11 @@ to
...
-```
\ No newline at end of file
+```
+
+Secrets:
+* [REQUIRED] `SNYK_TOKEN` - one can be inherited from the organisation - contact shaun.hare@dvsa.gov.uk or a DVSA member of your team to get that enabled.
+
+Inputs:
+* [REQUIRED] `php-version` - list of strings of all php versions which the workflow will run against
+* [OPTIONAL] `fail-fast` - boolean to control whether all steps in the matrix should be run, even if one of them fails
```
Comment (sdh100shaun) [inline README.md:564]: missing php library only

Comment (DariuszJerzewski) [reply README.md:564]: .github/workflows/php-tests.yml also uses matrix strategy and I've added `fail-fast` to both.

**Review by JoshuaLicense — APPROVED** [review]

**Review by sdh100shaun — APPROVED** [review]

**Pull request merged**